test/env.js

```diff
@@ -248,3 +248,67 @@ describe("Env", function() {
         });
     });
 });
+
+describe("#getRipeConfigOptions()", function() {
+    it("should return ci option for 'ci', 'test', 'now', 'sandbox', and 'stage' environments", function() {
+        const ciEnvOptions = ripeCommons.getRipeConfigOptions("ci");
+        const testEnvOptions = ripeCommons.getRipeConfigOptions("test");
+        const nowEnvOptions = ripeCommons.getRipeConfigOptions("now");
+        const sandboxEnvOptions = ripeCommons.getRipeConfigOptions("sandbox");
+        const stageEnvOptions = ripeCommons.getRipeConfigOptions("stage");
+        assert.deepStrictEqual(ciEnvOptions, {
+            url: "https://master--ripe-config.netlify.app/"
+        });
+        assert.deepStrictEqual(testEnvOptions, {
+            url: "https://master--ripe-config.netlify.app/"
+        });
+        assert.deepStrictEqual(nowEnvOptions, {
+            url: "https://master--ripe-config.netlify.app/"
+        });
+        assert.deepStrictEqual(sandboxEnvOptions, {
+            url: "https://master--ripe-config.netlify.app/"
+        });
+        assert.deepStrictEqual(stageEnvOptions, {
+            url: "https://master--ripe-config.netlify.app/"
+        });

```

Comment:
Even though they are all the same (for now) keep the same approach as the other tests suites of one test per environment.

Also add a test for the fallback.

Reply:
We have multiple per environment https://github.com/ripe-tech/ripe-commons-js/blob/d639b25ed15b83d604e9bc7711ee1bf311286761/test/env.js#L7-L17
Pretty much one for every environment we have

Reply:
You are right, missed that, thx!

Reply:
Add the fallback tests tho